cypress/integration/Trades.spec.js

```diff
@@ -1,120 +1,90 @@
 import { visitWithWeb3, tid, revertToSnapshot } from "../utils";
+import Trade from "../pages/Trade";
 
-context('Selling', () => {
-  const waitForTradeToFinish = 20000;
-
-  beforeEach(() => visitWithWeb3());
-  afterEach(() => revertToSnapshot());
+const waitForTradeToFinish = 20000;
 
-  it  ("ETH for ERC20 without PROXY", () => {
-    const base = "ETH";
-    const quote = "DAI";
-    const ETH_AMOUNT_TO_SELL = '1';
-    const DAI_AMOUNT_TO_RECEIVE = '280';
-    const expectedPrice = `280 ${base}/${quote}`;
+const newTrade = () => {
+  cy.get(tid("new-trade")).click({timeout: waitForTradeToFinish});
+};
 
+context('Selling', () => {
+  beforeEach(() => {
+    visitWithWeb3();
     cy.get(tid("wallets-continue")).contains("Continue").click();
+  });
+  afterEach(() => revertToSnapshot());
 
-    cy.get(tid("set-trade-from-amount"))
-      .find('input').type(ETH_AMOUNT_TO_SELL);
-
-    cy.get(tid("set-trade-to-amount"), {timeout: 2000})
-      .find('input').should('have.value', `${DAI_AMOUNT_TO_RECEIVE}.00000`);
-
-    //  NOTE: we need to click this exact spot to avoid downloading PDF file instead (link is in the button as well)
-    cy.get(tid("terms-and-conditions")).click({position: "topRight", force: true});
-    cy.get(tid("initiate-trade")).click();
-
-    cy.get(tid("trade-with-builtin-proxy-creation"))
-      .find('.details')
-      .find('.label.vertical-align span')
-      .then((value) => {
-        expect(value.text().trim()).to.eq('Create Proxy');
-      });
-
-    cy.get(tid("trade-token-from"))
-      .find(tid("token-amount-value"))
-      .then((value) => {
-        expect(value.text().trim()).to.eq(`${ETH_AMOUNT_TO_SELL} ${base}`);
-      });
+  it("ETH for ERC20 without PROXY", () => {
+    const from = 'ETH',
```

Comment:
can we agree on avoiding multiple assisnments with one const? Just repeat const multiple times like:
```
const from = 'ETH';
const to = 'DAI';
...
```

All styleguides that I know advice to do so: https://github.com/airbnb/javascript#variables--one-const